HEN_HOUSE/doc/src/pirs898-egs++/geometry.doxy

```diff
@@ -370,30 +368,93 @@ of the geometry definition. If the scene rendered by the viewer looks
+\endverbatim
+
+Alternatively, load the ptracks file using \c File-\>Open \c tracks.... The range
+of tracks to view can be limited using the two input boxes next to each particle
+type. Note that you can also load previously saved settings from the command line
+by also listing the .egsview file, before or after the .ptracks file.
+
+The dose, if scored to a 3ddose file, can be loaded using \c File-\>Open \c dose....
+If a dose scoring ausgab object (\ref EGS_DoseScoring) that is configured to
+produce a 3ddose file exists in the input file, then the 3ddose file will
+be automatically loaded and populate the list in the \c Dose section. The
+transparency of the dose is adjusted with the slider. If a 3ddose file is listed
+but disabled (or "grayed out"), this means that the 3ddose file dimensions
+did not match an \ref EGS_XYZGeometry in the input file.
+
+\image html egs_view1.png "The egs_view display tab"
+
+\subsection egs_view_colors The egs_view colors tab
+Various colors in the viewer can be controlled on this tab. Most importantly,
+material transparencies can be adjusted. Select the material of interest,
+set its color with the \c Color button, and set its transparency with the slider.
+
+Setting the other view colors should be intuitive. The particle tracks also
+have a checkbox to toggle energy-dependent transparency. Enabled by default,
+this option allows the transparency of track lines to be adjusted depending on
+the energy of the particle. Particles with the highest energy will not be
+transparent at all, and the lowest energy particles will be the most transparent.
+
+\image html egs_view2.png "The egs_view colors tab"
+
+\subsection egs_view_regions The egs_view regions tab
+The \c Regions tab allows the user to enable or disable the display of specific
+regions in the geometry. It does not actually remove the region from the geometry,
+so it can still be found by hovering the mouse. Rather, it sets the transparency
+of the region to 0. It also lists the material and color of every region.
+Of course, for complex geometries this list would be prohibitively long
+and affect the performance of the application, so
+the tab is disabled if greater than 1000 regions are in the geometry.
+
+\image html egs_view3.png "The egs_view regions tab"
+
+\subsection egs_view_camera The egs_view camera tab
+The \c Camera tab allows the user to control the camera position. A few camera
+presets are provided to look along each axis. To set a default view, position
+the camera as you like and click the \c Set \c Home button.
+
+An alternative way to control the view is through directly setting the
+\c Camera \c position. This sets where you are looking from, in the coordinates
+of the geometry. However, the viewer has a limitation that the \c Rotation \c point
+must reside within the geometry volume. To maintain a constant distance
+between the rotation point and the camera position, and camera position changes
```

Comment:
Is this instructing the user to  change the position of the rotation point if they change the camera position?  If so, the wording isn't totally clear.

Reply:
Thanks, I rephrased this.